### PR TITLE
Feature/initialize db

### PR DESCRIPTION
**Pull request by muhwagwa**
- [ ] Add `initialize-ott` API in ott/view.py
- [ ] Add `initialize-genre` API in content/view.py
- [ ] Add `initialize-contents` API in content/view.py

TODO (later on)
- add more set-up contents for initialize-contents API

### CODE REVIEW

File: backend/content/urls.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 from . import views
 
 urlpatterns = [
+    path('initialize/genre/', views.initialize_genre),
+    path('initialize/contents/', views.initialize_contents),
```
Comment (arclic) [inline backend/content/urls.py:6]: @muhwagwa I think making a new url is not a good idea, how about making a new separate file and execute once?

File: backend/content/views.py [hunk 2]
```diff
@@ -48,6 +47,80 @@ def request_the_movie_api(url, _params, max_retries=2, sleep_time=5):
...
+        Genre.objects.create( name = 'History' )
+        Genre.objects.create( name = 'Horror' )
+        Genre.objects.create( name = 'Music' )
+        Genre.objects.create( name = 'Mystery' )
+        Genre.objects.create( name = 'Romance' )
+        Genre.objects.create( name = 'Science Fiction' )
+        Genre.objects.create( name = 'TV Movie' )
+        Genre.objects.create( name = 'Thriller' )
+        Genre.objects.create( name = 'War' )
+        Genre.objects.create( name = 'Western' )
```
Comment (arclic) [inline backend/content/views.py:72]: @muhwagwa maybe we can use array

File: backend/ott/urls.py [hunk 3]
```diff
@@ -3,6 +3,7 @@
 
 urlpatterns = [
     # Ott
+    path('initialize/', views.initialize_ott),
```
Comment (arclic) [inline backend/ott/urls.py:6]: @muhwagwa same as above

File: backend/ott/views.py [hunk 4]
```diff
@@ -3,6 +3,41 @@
...
+            )
+        Ott.objects.create(
+            ott='CoupangPlay', membership='Basic', max_people=5, cost=2900, image='../ott_image/watcha_logo_ZmSSD6P.jpeg'
+            )
+        Ott.objects.create(
+            ott='Wavve', membership='Standard', max_people=2, cost=10900, image='../ott_image/watcha_logo_ZmSSD6P.jpeg'
+            )
+        Ott.objects.create(
+            ott='Wavve', membership='Premium', max_people=4, cost=12900, image='../ott_image/watcha_logo_ZmSSD6P.jpeg'
+            )
```
Comment (arclic) [inline backend/ott/views.py:39]: @muhwagwa same as above. I think making a default value set and add it in a separate file would be good

**Review by arclic — COMMENTED** [review]
@muhwagwa I have reviewed!

**Review by cw10108 — APPROVED** [review]

**Pull request merged**